dashboard/views.py

```diff
@@ -63,6 +64,9 @@ def home(request):
 def files(request):
     return render_to_response("files.html")
 
+def view_file_access_within_week(request):
```

Comment:
This looks like it could just use TemplateView in the urls

Reply:
Done!